README.md

```diff
@@ -91,6 +94,23 @@ It can:
 - Revoke the pending guardian (which means it can revoke any attempt to change the guardian).
 - Revoke the pending cap of any market.
 
+### Idle Supply
+
+In some cases, the vault's curator or allocators may want to keep some funds "idle", to guarantee lenders that some liquidity can be withdrawn from the vault (beyond the liquidity of each of the vault's markets).
+
+To achieve this, it is advised to allocate "idle" funds to any market on Morpho Blue having:
+
+- The vault's asset as loan token.
+- No collateral token (`address(0)`).
+- An arbitrary IRM.
+- No oracle (`address(0)`).
+- An arbitrary LLTV.
+
+Thus, these funds cannot be borrowed on Morpho Blue and are guaranteed to be liquid ; though it won't generate interest.
+
+Note that to allocate funds to this idle market, it is first required to enable its cap on MetaMorpho.
```

Comment:
```suggestion
Note that to allocate funds to this idle markets, it is first required to enable its cap on MetaMorpho.
```

Reply:
I don't get why we'd use a plural market ^^